pkg/teeth/publish_logs_test.go

```diff
@@ -124,3 +168,122 @@ ORDER BY
+					PullRequestURL: "https://github.com/foo/bar/pull/1",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+			},
+		},
+		{
+			name:     "success_with_batching",
+			bqClient: FakeBigQueryClientWithDefaultConfig(make([]*InvocationCommentStatusRecord, 0), nil, 2, DefaultBQInsertBatchWait),
+			statuses: []*InvocationCommentStatusRecord{
+				{
+					PullRequestID:  123456789,
+					PullRequestURL: "https://github.com/foo/bar/pull/1",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+				{
+					PullRequestID:  987654321,
+					PullRequestURL: "https://github.com/foo/bar/pull/2",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+				{
+					PullRequestID:  999999999,
+					PullRequestURL: "https://github.com/foo/bar/pull/3",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+			},
+		},
+		{
+			name:     "insert_fails",
+			bqClient: FakeBigQueryClientWithDefaultConfig(make([]*InvocationCommentStatusRecord, 0), errors.New("BOOM"), DefaultBQInsertBatchSize, DefaultBQInsertBatchWait),
+			statuses: []*InvocationCommentStatusRecord{
+				{
+					PullRequestID:  123456789,
+					PullRequestURL: "https://github.com/foo/bar/pull/1",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+			},
+			wantErrorCount: 1,
+		},
+		{
+			name:     "insert_fails_with_batching",
+			bqClient: FakeBigQueryClientWithDefaultConfig(make([]*InvocationCommentStatusRecord, 0), errors.New("BOOM"), 2, DefaultBQInsertBatchWait),
+			statuses: []*InvocationCommentStatusRecord{
+				{
+					PullRequestID:  123456789,
+					PullRequestURL: "https://github.com/foo/bar/pull/1",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+				{
+					PullRequestID:  987654321,
+					PullRequestURL: "https://github.com/foo/bar/pull/2",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+				{
+					PullRequestID:  999999999,
+					PullRequestURL: "https://github.com/foo/bar/pull/3",
+					ProcessedAt:    time.Now(),
+					CommentID:      bigquery.NullInt64{Int64: time.Now().Unix()},
+					Status:         "SUCCESS",
+					JobName:        "job-0",
+				},
+			},
+			wantErrorCount: 2,
+		},
+	}
+	for _, test := range tests {
```

Comment:
Prefer `tc` over `test` for this variable name

Reply:
Done.